### ISSUE TITLE
Actionother not detecting private messages (auto respond)

### ISSUE THREAD

**Issue by ipadalert** [ORelio/Minecraft-Console-Client#739]
Just to hop straight into it, basically I want to make my alt respond to private messages so I can respond to the person messaging my alt with, for example, /pay $1 $2 in the respond section of the match. However, when receiving messages from other players only actionprivate works and not actionother, which of course makes sense but I would like actionother to be able to detect these messages for the soul purpose of detecting the person who sent the message.
To claify further with images:
![Screen Shot 2019-05-25 at 6 03 26 PM](https://user-images.githubusercontent.com/45241936/58374995-68c18880-7f17-11e9-8ab1-0e3a753f183a.png)
![Screen Shot 2019-05-25 at 6 04 04 PM](https://user-images.githubusercontent.com/45241936/58375001-80990c80-7f17-11e9-80dc-17f2ec7590cd.png)
To sum this up, I want actionother to be able to detect private messages instead of actionprivate so I can use the ign of the person in the respond section of the [Match].

**Comment by ipadalert** [comment]
Figured it out nvm

**Comment by Mhowser** [comment]
Would you mind sharing your solution so others who might have the
same issue?

**Comment by ORelio** [comment]
The issue here was with grouping. First capturing group must contain username, and second group must contain message. The regex contains `(ipadoutpost|MV3|dacatttack)` which captures one of these three words instead of the player username : https://regex101.com/r/uRb3jX/1

Instead, use a non-capturing group `(:?ipadoutpost|MV3|dacatttack)` : https://regex101.com/r/uRb3jX/2

On a side note, the regex defined in config should not look for a specific username and a specific message. You have bot owners and autorespond for doing that.
